src/components/dar_collection_table/ResearcherActions.js

```diff
@@ -43,52 +41,24 @@ const hoverPrimaryButtonStyle = {
   color: 'white'
 };
 
-//Function to determine if collection is revisable
-//Should only show up if all of the DARs have a canceled status
-const isCollectionRevisable = (dars = {}) => {
-  const elections = flow(
-    flatMap(dar => !isEmpty(dar.elections) ? Object.values(dar.elections) : [])
-  )(dars);
-  return isEmpty(elections) && allCanceledDars(dars);
-};
-
-//Function to determine if collection is cancelable
-//Should only show up if none of the DARs have elections and not all DARs in the collection are canceled
-const isCollectionCancelable = (dars = {}) => {
-  const hasDars = !isEmpty(dars);
-  const hasNoElections = flow(
-    filter(dar => lowerCase(dar.data.status) !== 'canceled'),
-    map(dar => dar.elections),
-    flatMap((electionMap = {}) => Object.values(electionMap)),
-    isEmpty
-  )(dars);
-  return hasDars && !allCanceledDars(dars) && hasNoElections;
-};
-
-const allCanceledDars = (dars = {}) => {
-  return every(dar => lowerCase(dar.data.status) === 'canceled')(dars);
-};
-
 export default function ResearcherActions(props) {
   const { collection, showConfirmationModal } = props;
   const collectionId = collection.darCollectionId;
-  const { dars } = collection;
-  const history = useHistory();
 
-  const [cancelEnabled, setCancelEnabled] = useState(false);
-  const [reviseEnabled, setReviseEnabled] = useState(false);
+  const uniqueId = (collectionId ? collectionId : collection.referenceIds[0]);
 
-  useEffect(() => {
-    const isCancelable = isCollectionCancelable(dars);
-    const isRevisable = isCollectionRevisable(dars);
-    setCancelEnabled(isCancelable);
-    setReviseEnabled(isRevisable);
-  }, [dars]);
+  const history = useHistory();
+
+  const cancelEnabled = collection.actions.includes('Cancel');
+  const reviseEnabled = collection.actions.includes('Revise');
+  const reviewEnabled = collection.actions.includes('Review');
+  const deleteEnabled = collection.actions.includes('Delete');
+  const resumeEnabled = collection.actions.includes('Resume');
```

Comment:
Optional suggestion. It would be nice if these string constants were part of an object/enum somewhere useful for all action-related components. Not sure where that would be for now, so definitely not a blocker for this PR.